Portal/src/Datahub.Portal/Pages/Project/DataProject/DataProjectPage.razor

```diff
@@ -38,6 +38,10 @@
             <MudItem sm="12" md="5" lg="4">
                 <ProjectUserCardList ProjectAcronym="@ProjectAcronymParam"/>
             </MudItem>
+
```

Comment:
We need the enabled flag to display this one - otherwise all the workspaces will be affected

Reply:
We have to setup something in the Project entry to know there is an app associated.

Reply:
Done!